components/navigation/topMenu/topMenu.tsx

```diff
@@ -39,6 +51,9 @@ export function TopMenu() {
   function alertLink(url: string) {
     alert(url);
```

Comment:
Do we want to actually display an alert? I see an issue where when dismissing the share sheet, an alert still appears.

Reply:
@George-Zhang5246